--- src/amdgpu_present.c
+++ src/amdgpu_present.c
@@ -292,27 +292,12 @@
 		return FALSE;
 	}
 
-	/* Only DC supports advanced color management features, so we can use
-	 * drmmode_cm_enabled as a proxy for "Is DC enabled?"
+	/* The kernel driver doesn't handle flipping between BOs with different
+	 * tiling parameters correctly yet
 	 */
-	dc_enabled = drmmode_cm_enabled(&info->drmmode);
-
-	if (info->dri2.pKernelDRMVersion->version_minor < (dc_enabled ? 31 : 34)) {
-		/* The kernel driver doesn't handle flipping between BOs with
-		 * different pitch correctly
-		 */
-		if (pixmap->devKind != screen_pixmap->devKind)
-			return FALSE;
-	}
-
-	if (!dc_enabled || info->dri2.pKernelDRMVersion->version_minor < 31) {
-		/* The kernel driver doesn't handle flipping between BOs with
-		 * different tiling parameters correctly
-		 */
-		if (amdgpu_pixmap_get_tiling_info(pixmap) !=
-		    amdgpu_pixmap_get_tiling_info(screen_pixmap))
-			return FALSE;
-	}
+	if (amdgpu_pixmap_get_tiling_info(pixmap) !=
+		amdgpu_pixmap_get_tiling_info(screen_pixmap))
+		return FALSE;
 
 	for (i = 0, num_crtcs_on = 0; i < config->num_crtc; i++) {
 		if (drmmode_crtc_can_flip(config->crtc[i]))
